### PR TITLE
migrate drone to coveralls and fixed some test cases

### PR DESCRIPTION
**Pull request by syedowaisali90**
Hi @ndegwamartin I have migrated drone ci to coveralls and fixed some test cases

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Install NDK
+        run: echo "y" | sudo ${ANDROID_HOME}/tools/bin/sdkmanager --install "ndk;21.0.6113669" --sdk_root=${ANDROID_SDK_ROOT}
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Run unit tests with Gradle
+        run: ./gradlew jacocoTestReport --stacktrace
+      - name: Generate Javadoc with Gradle
```
Comment (ndegwamartin) [inline .github/workflows/ci.yml:29]: Java docs is not set up for this repo, could we remove this step?

Comment (syedowaisali90) [reply .github/workflows/ci.yml]: removed.

**Comment by ndegwamartin** [comment]
@syedowaisali90 if we need a new release from this please remember to update the _Version Code_ and _Version Name_ 

[here](https://github.com/onaio/steps-app/blob/8d677ff08d8a5a17b621b2f1862a2cd34e37fcb2/build.gradle#L22)

**Comment by syedowaisali90** [comment]
> @syedowaisali90 if we need a new release from this please remember to update the _Version Code_ and _Version Name_
> 
> [here](https://github.com/onaio/steps-app/blob/8d677ff08d8a5a17b621b2f1862a2cd34e37fcb2/build.gradle#L22)

sure

**Pull request merged**